playwright.config.ts

```diff
@@ -0,0 +1,62 @@
+import path from 'node:path';
+import { defineConfig, devices } from '@playwright/test';
+import { authFile } from './tests/fixtures/helpers';
+
+if (!process.env.CI) {
+  // eslint-disable-next-line @typescript-eslint/no-require-imports
+  require('dotenv').config({ path: path.join(__dirname, 'tests', '.env') });
+}
+
+export default defineConfig({
+  testDir: './tests',
+  // We don't want this set to true as that would make tests in each file to run
+  // in parallel, which will cause conflicts with the "global state". With this
+  // set to false and workers > 1, multiple test files can run in parallel, but
+  // tests within a file are run at one at a time. We make extensive use of
+  // worker-scope fixtures and beforeAll hooks to achieve best performance.
+  fullyParallel: false,
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 2 : 0,
+  workers: process.env.CI ? 1 : undefined,
+  reporter: [['list'], ['html', { open: 'never' }]],
+  use: { trace: 'on-first-retry' },
+
+  projects: [
+    { name: 'setup', testMatch: /.*\.setup\.ts/ },
+
+    {
+      name: 'chromium',
+      use: { ...devices['Desktop Chrome'], storageState: authFile },
+      dependencies: ['setup'],
+    },
+
+    // {
+    //   name: 'firefox',
+    //   use: { ...devices['Desktop Firefox'], storageState: authFile },
+    //   dependencies: ['setup'],
+    // },
+
+    // {
+    //   name: 'webkit',
+    //   use: { ...devices['Desktop Safari'], storageState: authFile },
+    //   dependencies: ['setup'],
+    // },
+
+    /* Test against branded browsers. */
+    // {
+    //   name: 'edge',
+    //   use: { ...devices['Desktop Edge'], channel: 'msedge' },
+    // },
+    // {
+    //   name: 'chrome',
+    //   use: { ...devices['Desktop Chrome'], channel: 'chrome' },
+    // },
+  ],
+
+  /* Run your local dev server before starting the tests */
+  // webServer: {
+  //   command: 'npm run start',
+  //   url: 'http://127.0.0.1:3000',
+  //   reuseExistingServer: !process.env.CI,
+  // },
```

Comment:
This can be used to setup a mock server if we do not want to use the Test Wallet during testing?

Reply:
Exactly